ethexe/contracts/test/Router.t.sol

```diff
@@ -11,16 +11,16 @@ contract RouterTest is Base {
     using MessageHashUtils for address;
     using FROSTOffchain for SigningKey;
 
-    address immutable deployer = 0x116B4369a90d2E9DA6BD7a924A23B164E10f6FE9;
+    address public deployer = 0x116B4369a90d2E9DA6BD7a924A23B164E10f6FE9;
```

Comment:
Could you explain please why not immutable now or it does not matter cause it's tests?

Reply:
I don't see any point in saving gas for storage in tests